src/metadata.rs

```diff
@@ -794,6 +794,29 @@ mod test {
         let content = metadata.to_file_contents();
         let pkginfo: Result<python_pkginfo::Metadata, _> = content.parse();
         assert!(pkginfo.is_ok());
+
+        let license = metadata
+            .to_vec()
+            .into_iter()
+            .filter_map(|(key, val)| {
+                if key.starts_with("License") {
+                    Some(val)
+                } else {
+                    None
+                }
+            })
+            .next()
+            .unwrap();
+        let mut lines = license.split_inclusive('\n');
+        assert_eq!(
+            lines.next().unwrap(),
+            "Copyright (c) 2018-present konstin\r\n"
```

Comment:
Without `trim_end()`, this becomes `Copyright (c) 2018-present konstin\r\r\n` on Windows.

https://github.com/messense/maturin/runs/5465287747?check_suite_focus=true